evap/staff/templates/staff_course_contributor_import.html

```diff
@@ -0,0 +1,21 @@
+{% extends "staff_course_base.html" %}
+
+{% load static %}
+{% load bootstrap3 %}
+
+{% block content %}
+    {{ block.super }}
+    <p>
+        {% trans "Copy contributor data from another course or import from Excel file" %} (<a href="{% get_static_prefix %}sample_user.xls">{% trans "Sample File"%}</a>). {% trans "The latter will create user accounts for all contained coaches if necessary before adding them as contributors for the course." %}
```

Comment:
I just copy pasted it and didn't see it (silly soft wraps). I also changed it in the original file now.